### DISCUSSION TITLE
How to server content from mitm.it?

### DISCUSSION THREAD

**Discussion by bra-fsn** [mitmproxy/mitmproxy#5329] [General]
During implementing a proxy in mitmproxy I wanted to test whether it works correctly or not, but there are many variables which I need to sync between the proxy and the test app.
The best way would be to serve the configuration variables through the proxy itself, for which the site http://mitm.it/ seems to be straightforward (where the certs are served).
So the question is: is it possible to serve a URL from mitmproxy itself, which can be accessible only through the proxy under the domain mitm.it?
I could use the example wsgi-flask-app.py to achieve the same with another domains, but I'm curious if it's possible to do this the already "captured" mitm.it?

**Comment by mhils** [comment]
Extending mitm.it is not encouraged, the `wsgi-flask-app.py` example does indeed show what we recommend.
(mitm.it is implemented the same way, see https://github.com/mitmproxy/mitmproxy/blob/main/mitmproxy/addons/onboarding.py)
